tol_lab_share/processors/create_labware_processor.py

```diff
@@ -20,8 +20,20 @@ def __init__(self, schema_registry, basic_publisher, config):
 
     def process(self, message: RabbitMessage) -> bool:
         logger.debug("CreateLabwareProcessor::process")
-        barcode = message.message["barcode"]
-        encoded_message = self._encoder.encode([{"success": f"ok create for barcode: { barcode }"}])
+        logger.debug(f"Received: { message.message }")
+
+        message = {
+            "sourceMessageUuid": str(message.message["messageUuid"].decode("utf-8")),
```

Comment:
You needn't specify `"utf-8"` during decode.  It's the default value for the encoding type.

Reply:
Ok